recipe/meta.yaml

```diff
@@ -1,35 +1,88 @@
-{% set name = "openfst" %}
 {% set version = "1.8.2" %}
 
 package:
-  name: {{ name|lower }}
+  name: openfst
   version: {{ version }}
 
 source:
-  url: http://www.openfst.org/twiki/pub/FST/FstDownload/{{ name }}-{{ version }}.tar.gz
+  url: http://www.openfst.org/twiki/pub/FST/FstDownload/openfst-{{ version }}.tar.gz
   sha256: de987bf3624721c5d5ba321af95751898e4f4bb41c8a36e2d64f0627656d8b42
   patches:
-    - osx_arm64_configure.patch  # [build_platform != target_platform]
+    - patches/0001-Remove-float-equality-check-in-cross-compiling.patch  # [build_platform != target_platform]
+    - patches/0002-add-missing-header.patch
+    - patches/0003-add-no-undefined-to-some-LDFLAGS.patch                # [win]
+    - patches/0004-Fix-file-closing-on-windows.patch                     # [win]
 
 build:
-  number: 1
-  skip: true  # [win]
+  number: 2
+  run_exports:
+  # seems to be changing even with patch releases
```

Comment:
I don't know what this stanza does. Could you document?

Reply:
It makes sure that any library using openfst as a host dependence (i.e. compiling against the shared library) gains the corresponding runtime dependence. It's part of the standard tooling for shared libs in conda-forge

Reply:
https://conda-forge.org/docs/maintainer/pinning_deps.html#specifying-run-exports

Reply:
I think that information should go into a comment here.

Reply:
Ping on this: I am not going to remember what any of this means without some comments.